srcpkgs/python3-lazy-object-proxy/template

```diff
@@ -11,11 +11,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-2-Clause"
 homepage="https://github.com/ionelmc/python-lazy-object-proxy"
 distfiles="${PYPI_SITE}/l/lazy-object-proxy/lazy-object-proxy-${version}.tar.gz"
-checksum=f3900e8a5de27447acbf900b4750b0ddfd7ec1ea7fbaf11dfa911141bc522af0
-
-post_patch() {
-	vsed -i '/setuptools_scm/d' setup.cfg
-}
```

Comment:
This was an attempt to stop `setuptools` from fetching dependencies at build time (see https://travis-ci.org/github/void-linux/void-packages/jobs/735601759#L570), but I missed the bad versioning in the `egg-info`. When you drop this patch, please replace the `python3-setuptools` in `hostmakedepends` with `python3-setuptools_scm`, which I added to address this problem. (That package depends on `python3-setuptools`.)

Also, it's a good idea to check whether `${wrksrc}/.eggs` exists after `xbps-src build` and, if so, what its contents are. These are eggs fetched by `setuptools` during build. This is bad, because it diminishes reproducibility, makes the build process more fragile, and is deprecated.

A couple of packages that use `setuptools_scm` also use `setuptools_scm_git_archive`, which doesn't seem to be useful in versioning. That can probably be patched out without breaking the package; see [`srcpkgs/python3-Cheroot/template`](https://github.com/void-linux/void-packages/tree/master/srcpkgs/python3-Cheroot/template).

Reply:
Thanks for the detailed response and explanation.  I have made the change to `hostmakedepends` and will look out for the `${wrksrc}/.eggs` in the future.